src/main/resources/logging_info.properties

```diff
@@ -16,8 +16,7 @@
 # By default we only configure a ConsoleHandler, which will only
 # show messages at the INFO and above levels.
 #handlers= java.util.logging.ConsoleHandler
-handlers= java.util.logging.FileHandler, java.util.logging.ConsoleHandler
-
+handlers = org.slf4j.bridge.SLF4JBridgeHandler
```

Comment:
Just one note here. If someone is dependent on that logs are written to a file I think we need to add an optional filehandler in logback.xml.